examples/glfw/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.15)
+project(FlutterEmbedderGLFW)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11" )
+
+add_executable(flutter_glfw FlutterEmbedderGLFW.cc)
+
+############################################################
+# GLFW
+############################################################
+find_path(GLFW_INCLUDE_PATH "glfw3.h" /usr/local/Cellar/glfw/3.3/include/GLFW/)
+include_directories(${GLFW_INCLUDE_PATH})
+find_library(GLFW_LIB glfw /usr/local/Cellar/glfw/3.3/lib)
+target_link_libraries(flutter_glfw ${GLFW_LIB})
+
+############################################################
+# Flutter Engine
+############################################################
+# This is assuming you've built a local version of the Flutter Engine.  If you
+# downloaded yours is from the internet you'll have to change this.
+include_directories(${CMAKE_SOURCE_DIR}/../../shell/platform/embedder)
+find_library(FLUTTER_LIB flutter_engine PATHS ${CMAKE_SOURCE_DIR}/../../../out/host_debug_unopt)
```

Comment:
You can make the path a CMake option. Then its fine to default it to `host_debug_unopt`.

Reply:
FLUTTER_LIB is already a variable.  If I pull the engine name out into a different variable, editing it doesn't update FLUTTER_LIB on subsequent cmake calls.  People should just edit FLUTTER_LIB.